### PR TITLE
Update filelogreceiver/README.md

### LINKED ISSUE #2335: filelogreceiver - update readme
The readme is badly out of date and still reflects the old `stanzareceiver`.

### PR DESCRIPTION
**Pull request by djaglowski**
Resolves #2335

### CODE REVIEW

File: receiver/filelogreceiver/README.md [hunk 1]
```diff
@@ -1,29 +1,63 @@
...
 
-## Optional Parameters
+Note that _by default_, no logs will be read from a file that is not actively being written to because `start_at` defaults to `end`.
 
-- `plugin_dir` is the path to a directory which contains `stanza` [plugins](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/docs/plugins.md#using-plugins). Plugins are parameterized pipelines that are designed for specific use cases.
-- `offsets_file` is the path to a file that `stanza` will use to remember where it left off when reading from files or other persistent input sources. If specified, `stanza` will create and manage this file.
+### Operators
 
-## Operator Basics
+Each operator performs a simple responsibility, such as parsing a timestamp or JSON. Chain together operators to process logs into a desired format.
```
Comment (tigrannajaryan) [inline receiver/filelogreceiver/README.md:35]: Only "parser" and "general purpose" operators make sense to be used, right? Maybe mention that here.

File: receiver/filelogreceiver/README.md [hunk 2]
```diff
@@ -1,29 +1,63 @@
...
+| Field                  | Default          | Description                                                                                                        |
+| ---                    | ---              | ---                                                                                                                |
+| `include`              | required         | A list of file glob patterns that match the file paths to be read                                                  |
+| `exclude`              | []               | A list of file glob patterns to exclude from reading                                                               |
+| `start_at`             | `end`            | At startup, where to start reading logs from the file. Options are `beginning` or `end`                            |
+| `write_to`             | $                | The record [field](/docs/types/field.md) written to when creating a new log entry                                  |
```
[the remark this answers is not in the document]
Comment (djaglowski) [reply receiver/filelogreceiver/README.md:18]: `$` can be used as shorthand for `$record`, but this does not appear to be documented. 

In any case, I will just update this to the expanded form.

[the remark this answers is not in the document]
Comment (djaglowski) [reply receiver/filelogreceiver/README.md]: I'm also going to clean up the `fields` documentation a bit. Will link back here shortly.

[the remark this answers is not in the document]
Comment (djaglowski) [reply receiver/filelogreceiver/README.md]: https://github.com/open-telemetry/opentelemetry-log-collection/pull/21

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**